pandas/sparse/frame.py

```diff
@@ -187,9 +213,47 @@ def _init_matrix(self, data, index, columns, dtype=None):
         if len(index) != N:
             raise ValueError('Index length mismatch: %d vs. %d' %
                              (len(index), N))
+        return index, columns
 
-        data = dict([(idx, data[:, i]) for i, idx in enumerate(columns)])
-        return self._init_dict(data, index, columns, dtype)
+    def to_coo(self):
+        """
+        Convert the frame to its SciPy sparse COO matrix representation.
```

Comment:
There is no index (either top or side). A simple 2-D spmatrix (ndim==2) matrix is returned, laid out as the frame was. It's like `.values`, but for retaining sparsity.

What do I write?